test/benchmarks/src/main/scala/scala/RegexUnapplyBenchmark.scala

```diff
@@ -0,0 +1,85 @@
+package scala
```

Comment:
Let's move this to `scala.util.matching`

Reply:
done